core/base/inc/TROOT.h

```diff
@@ -270,8 +271,8 @@ friend TROOT *ROOT::Internal::GetROOT2();
    TObject          *GetGeometry(const char *name) const;
    const TObject    *GetSelectedPrimitive() const { return fPrimitive; }
    TVirtualPad      *GetSelectedPad() const { return fSelectPad; }
-   Int_t             GetNclasses() const { return fClasses->GetSize(); }
-   Int_t             GetNtypes() const { return fTypes->GetSize(); }
+   Int_t             GetNclasses() const;
```

Comment:
Why make that change?

Reply:
Because `TList.h` is not included with `R__LESS_INLCUDES` - see my previous comment

Reply:
I see.  This illustrate a trade off:  small speedup at parsing time (removing the the include of TList.h for the (rare?) file that uses TROOT but not TList) vs a small performance increase (for the code that uses GetNclasses and GetNtypes).

Whether one is better than the other is  .. well .. hard to know (and in probably in both case the actual performance change is likely indistinguishable) .. I would probably not have made that change but reverting it is also probably not worth it.

Reply:
@pcanal Should I revert these changes in TROOT.h?

Reply:
Up to you in the end.  I slightly lean for revert but not if it is too much hassle.

Reply:
I prefer to keep changes. These methods are not crucial for performance.